libs/SalesforceSDK/src/com/salesforce/androidsdk/security/PasscodeManager.java

```diff
@@ -75,14 +75,14 @@
     // Key in preference for the passcode length.
     private static final String KEY_PASSCODE_LENGTH = "passcode_length";
 
-    // Request code used to start passcode activity
-    public static final int PASSCODE_REQUEST_CODE = 777;
-
-    // Key used to specify that a longer passcode needs to be created.
-    public static final String CHANGE_PASSCODE_KEY = "change_passcode";
```

Comment:
Passcode activity asks passcode manager directly - no longer passing the flag through the intent.